<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogDbPurchases</class>
 <widget class="QDialog" name="DialogDbPurchases">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>922</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="lineWidth">
      <number>4</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QTableView" name="tableViewData">
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="QGroupBox" name="groupBoxLog">
      <property name="title">
       <string>Статус</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPlainTextEdit" name="plainTextEditLog"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxSettings">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Настройки</string>
     </property>
     <widget class="QLabel" name="labelDescViewMode">
      <property name="geometry">
       <rect>
        <x>4</x>
        <y>24</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Показывать</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxViewMode">
      <property name="geometry">
       <rect>
        <x>69</x>
        <y>22</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelUnitViewMode">
      <property name="geometry">
       <rect>
        <x>136</x>
        <y>24</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>продуктов</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxDateTimeUsage">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>45</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonDeleteRow">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить строку</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonAppendRow">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить строку</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelDescDateViewMode">
      <property name="geometry">
       <rect>
        <x>4</x>
        <y>50</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Формат даты</string>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="dateTimeEditMapper">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>181</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelDescPurchaseDate">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Дата заказа</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
